webapp/src/shared/components/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -1,5 +1,6 @@
 import Text, { TextColor, TextVariant, TextWeight } from '../Text/Text';
 import './ToggleSwitch.css';
+import React from 'react';
```

Comment:
I'm considering making a 'disabled' version of the toggle, so that I can expressively show if a parameter can or cannot be edited